main.lua

```diff
@@ -0,0 +1,104 @@
+imgxKaizer=200
+imgyKaizer=0
+
+imgxC1 = 0
+imgyC1 = 100
+
+imgxC2 = 0
+imgyC2 = 200
+
+imgxC3 = 0
+imgyC3 = 300
+
+function love.load()
+    imageKaizer = love.graphics.newImage("Kaizer.JPG")
+    widthKaizer = imageKaizer:getWidth()
+    heightKaizer = imageKaizer:getHeight()    
+    
+    imageCar1 = love.graphics.newImage("Car1.JPG")
+    widthCar1 = imageCar1:getWidth()
+    heightCar1 = imageCar1:getHeight()
+
+    imageCar2 = love.graphics.newImage("Car2.JPG")
+    widthCar2 = imageCar2:getWidth()
+    heightCar2 = imageCar2:getHeight()
+
+    imageCar3 = love.graphics.newImage("Car3.JPG")
+    widthCar3 = imageCar3:getWidth()
+    heightCar3 = imageCar3:getHeight()

```

Comment:
inconsistent naming, on the top you used "C3" but here you're using "Car3"

Reply:
Renamed all instances of C1 C2 C3 to "Car 1" etc.